notebook/agentchat_qdrant_RetrieveChat.ipynb

```diff
@@ -0,0 +1,1220 @@
+    "\n",
+    "# 1. create an RetrieveAssistantAgent instance named \"assistant\"\n",
+    "assistant = RetrieveAssistantAgent(\n",
+    "    name=\"assistant\", \n",
+    "    system_message=\"You are a helpful assistant.\",\n",
+    "    llm_config={\n",
+    "        \"request_timeout\": 600,\n",
+    "        \"seed\": 42,\n",
+    "        \"config_list\": config_list,\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "# 2. create the QdrantRetrieveUserProxyAgent instance named \"ragproxyagent\"\n",
+    "# By default, the human_input_mode is \"ALWAYS\", which means the agent will ask for human input at every step. We set it to \"NEVER\" here.\n",
+    "# `docs_path` is the path to the docs directory. It can also be the path to a single file, or the url to a single file. By default, \n",
+    "# it is set to None, which works only if the collection is already created.\n",
+    "# \n",
+    "# Here we generated the documentations from FLAML's docstrings. Not needed if you just want to try this notebook but not to reproduce the\n",
+    "# outputs. Clone the FLAML (https://github.com/microsoft/FLAML) repo and navigate to its website folder. Pip install and run `pydoc-markdown`\n",
+    "# and it will generate folder `reference` under `website/docs`.\n",
+    "#\n",
+    "# `task` indicates the kind of task we're working on. In this example, it's a `code` task.\n",
+    "# `chunk_token_size` is the chunk token size for the retrieve chat. By default, it is set to `max_tokens * 0.6`, here we set it to 2000.\n",
+    "# We use an in-memory QdrantClient instance here. Not recommended for production.\n",
+    "# Get the installation instructions here: https://qdrant.tech/documentation/guides/installation/\n",
+    "ragproxyagent = QdrantRetrieveUserProxyAgent(\n",
+    "    name=\"ragproxyagent\",\n",
+    "    human_input_mode=\"NEVER\",\n",
+    "    max_consecutive_auto_reply=10,\n",
+    "    retrieve_config={\n",
+    "        \"task\": \"code\",\n",
+    "        \"docs_path\": \"~/code/FLAML/website/docs/reference\",  # change this to your own path, such as https://raw.githubusercontent.com/microsoft/autogen/main/README.md\n",
+    "        \"chunk_token_size\": 2000,\n",
+    "        \"model\": config_list[0][\"model\"],\n",
+    "        \"client\": QdrantClient(\":memory:\"),\n",
+    "        \"embedding_model\": \"BAAI/bge-small-en-v1.5\",\n",
+    "    },\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id=\"example-1\"></a>\n",
+    "### Example 1\n",
+    "\n",
+    "[back to top](#toc)\n",
+    "\n",
+    "Use RetrieveChat to answer a question and ask for human-in-loop feedbacks.\n",
+    "\n",
+    "Problem: Is there a function named `tune_automl` in FLAML?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
```

Comment:
The outputs doesn't seem to match the code.

Reply:
Committed the latest outputs.
https://github.com/microsoft/autogen/pull/303/commits/d09797d432002cba3d7ce6c8e1dbea228310ca77